src/extras/render-passes/render-pass-depth-aware-blur.js

```diff
@@ -0,0 +1,94 @@
+import { RenderPassShaderQuad } from '../../scene/graphics/render-pass-shader-quad.js';
+import { shaderChunks } from '../../scene/shader-lib/chunks/chunks.js';
+
+/**
+ * Render pass implementation of a depth-aware bilateral blur filter.
+ *
+ * @category Graphics
+ * @ignore
+ */
+class RenderPassDepthAwareBlur extends RenderPassShaderQuad {
+    constructor(device, sourceTexture) {
+        super(device);
+        this.sourceTexture = sourceTexture;
+
+        this.shader = this.createQuadShader('DepthAwareBlurShader', shaderChunks.screenDepthPS + /* glsl */`
```

Comment:
these should probably be overridable chunks?

Reply:
Not really. Making those chunks would stop their three-shaking. The render pass is a combination of shader and it's parameters, bot nothing else in the file.

If the users wants to use for example different depth aware blur, they can simply create a different render pass (shader + it's set up) and use that instead.